### ISSUE TITLE
Armor rendering problem

### ISSUE THREAD

**Issue by Omgise** [makamys/SmartMoving#3]
![2022-11-19_09 29 25](https://user-images.githubusercontent.com/106000018/202827995-b6c4284e-31e4-43c6-a413-8bfd60caadc2.png)
![2022-11-19_09 29 56](https://user-images.githubusercontent.com/106000018/202827997-e01faddf-94ff-467d-8249-ed2a164cc07a.png)

**Comment by Omgise** [comment]
There is also the rendering problem of Adventure Backpack
![2022-11-19_10 14 32](https://user-images.githubusercontent.com/106000018/202829723-53b5d32c-a97f-4e70-b891-cc3e6781bd00.png)
![2022-11-19_10 14 44](https://user-images.githubusercontent.com/106000018/202829728-3dc1448a-f94c-41d6-b5b0-e4abc37deacd.png)

**Comment by tyrael93** [comment]
Yes that's an old smart moving issue. Botania and several other types of armor from particular mods tend to do that, yeah. There's a few select armors that don't have hooks (?) and can't be rendered properly with smart moving.

I think the same issue applies to the backpack mod and also, as with lot of others (better storage etc), although I am pretty sure I remember the satchels mod does work with it. What I've been doing is whenever I'm using botania armor with smart moving I'll just hide it from rendering through armourer's workshop. It'd be nice if the issue could at least be pinpointed as to what is causing it in the first place though.

**Comment by makamys** [comment]
SmartMoving replaces the player model completely, so other mods that try to change the model get confused (there's an incomplete list [here](https://github.com/makamys/SmartMoving/wiki/Compatibility-List)). Satchels originally didn't work either, I had to rewrite its model renderer with Smart Moving in mind in order to make it render properly.

There seem to be many different ways in which mods modify the player model, so making them all compatible is rather difficult. One thing that could effectively solve the problem is if there was an option to make Smart Moving use a simpler animation system which would be implemented in a more lightweight way. I will look into that at some point. (This would also remove the dependency on SmartRender and RenderPlayerAPI.)
